lib/rubocop/cop/layout/empty_lines_around_access_modifier.rb

```diff
@@ -224,6 +228,8 @@ def should_insert_line_after?(node)
         end
 
         def inside_block?(node)
+          return false unless node.parent
+
```

Comment:
The test seems to pass even without this condition. Is this condition necessary?

```suggestion
```

Reply:
Yeah, it's obsolete now. Thanks.